specification/maps/data-plane/Route/preview/2024-06-01-preview/route.json

```diff
@@ -0,0 +1,3597 @@
+        "operationId": "Route_PostRouteMatrixAsync",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "operation-location",
+          "final-state-schema": "#/definitions/RouteAsyncOperation"
+        },
+        "consumes": [
+          "application/geo+json"
+        ],
+        "produces": [
+          "application/json"
```

Comment:
response content type might be "application/geo+json" only

Reply:
for  async response the result is json.